src/bin/shared/buildContext.ts

```diff
@@ -0,0 +1,876 @@
+                        "node_modules",
+                        ".cache"
+                    );
+                })(),
+                "keycloakify"
+            );
+
+            return cacheDirPath;
+        })(),
+        urlPathname: (() => {
+            webpack: {
+                if (resolvedViteConfig !== undefined) {
+                    break webpack;
+                }
+
+                const { homepage } = parsedPackageJson;
+
+                let url: URL | undefined = undefined;
+
+                if (homepage !== undefined) {
+                    url = new URL(homepage);
+                }
+
+                if (url === undefined) {
+                    return undefined;
+                }
+
+                const out = url.pathname.replace(/([^/])$/, "$1/");
+                return out === "/" ? undefined : out;
+            }
+
+            return resolvedViteConfig.urlPathname;
+        })(),
+        assetsDirPath: (() => {
+            webpack: {
+                if (resolvedViteConfig !== undefined) {
+                    break webpack;
+                }
+
+                if (buildOptions.staticDirPathInProjectBuildDirPath !== undefined) {
+                    getAbsoluteAndInOsFormatPath({
+                        pathIsh: buildOptions.staticDirPathInProjectBuildDirPath,
+                        cwd: projectBuildDirPath
+                    });
+                }
+
+                return pathJoin(projectBuildDirPath, "static");
+            }
+
+            return pathJoin(projectBuildDirPath, resolvedViteConfig.assetsDir);
+        })(),
+        kcContextExclusionsFtlCode: (() => {
+            if (buildOptions.kcContextExclusionsFtl === undefined) {
+                return undefined;
+            }
+
+            if (buildOptions.kcContextExclusionsFtl.endsWith(".ftl")) {
+                const kcContextExclusionsFtlPath = getAbsoluteAndInOsFormatPath({
+                    pathIsh: buildOptions.kcContextExclusionsFtl,
+                    cwd: projectDirPath
+                });
+
+                return fs.readFileSync(kcContextExclusionsFtlPath).toString("utf8");
+            }
+
+            return buildOptions.kcContextExclusionsFtl;
+        })(),
+        environmentVariables: buildOptions.environmentVariables ?? [],
+        recordIsImplementedByThemeType,
+        themeSrcDirPath,
+        fetchOptions: getProxyFetchOptions({
+            npmConfigGetCwd: (function callee(upCount: number): string {
+                const dirPath = pathResolve(
+                    pathJoin(...[projectDirPath, ...Array(upCount).fill("..")])
+                );
+
+                assert(
+                    dirPath !== pathSep,
+                    "Couldn't find a place to run 'npm config get'"
+                );
+
+                try {
+                    child_process.execSync("npm config get", {
+                        cwd: dirPath,
+                        stdio: "ignore"
```

Comment:
Setting this to `ignore` causes the stderr output not to be included in the thrown error which is why this code doesn't work as intended. Changing this to pipe fixes this issue without printing the output to the console.

```suggestion
                        stdio: "pipe"
```

Reply:
Applied!